DeepMenu/Dock/MuDock.swift

```diff
@@ -122,11 +122,12 @@ class MuDock: Identifiable, ObservableObject {
 
     func findHover(_ touchNow: CGPoint) -> MuPod? {
 
+        // not hovering over dock? 
         if !bounds.contains(touchNow) {
-            //?? return nil
+            //TODO: return nil has false positives
         }
 
-        // todo: this is rather inefficient
+        //TODO: this is rather inefficient, is a workcout for
```

Comment:
workcout?

Reply:
GitKraken doesn't allow an inline edit? should be "workaround for above"

Reply:
GitKraken doesn't but GitHub does so I edited it there